src/main/flight/servos.c

```diff
@@ -296,20 +296,54 @@ STATIC_UNIT_TESTED void forwardAuxChannelsToServos(uint8_t firstServoIndex)
     }
 }
 
+static void biquadFilterInitd1d2(biquadFilter_t *filter, float sample)
+{
+    filter->d2 = filter->b2 * sample - filter->a2 * sample;
+    filter->d1 = filter->b1 * sample - filter->a1 * sample + filter->d2;
+}
+
 static void filterServos(void)
 {
-    if (servoConfig()->servo_lowpass_freq) {
+    static uint16_t rudderFilterFreq;
```

Comment:
Presumably `rudderFilterFreq` actually means the tail servo? If so can you rename to `tailServoFilterFreq`? Since iNav is also used on airplanes, calling this `rudderFilterFreq` may cause confusion for future code maintainers.